### PR TITLE
Update installation to mamba

### LINKED ISSUE #222: Update to Mamba installation
As now requested we should change the installation lesson fomr Conda to Mamba.

### PR DESCRIPTION
**Pull request by LisaBock**
As now requested we will change the installation lesson fomr Conda to Mamba.

* * *

Closes #222

### CODE REVIEW

File: _episodes/02-installation.md [hunk 1]
```diff
@@ -57,59 +57,47 @@ in the [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installa
...
+
+    ```bash
+    mamba update --name base mamba
+    ```
+
+6.  Verify you have a working mamba installation by: 
 
     ```bash
-    conda list
+    which mamba
```
Comment (rswamina) [inline _episodes/02-installation.md:95]: Should this return a value in the terminal, like the path? We need to add that I think.

File: _episodes/02-installation.md [hunk 2]
```diff
@@ -221,19 +222,16 @@ installed in it.
...
 >         issue](https://github.com/ESMValGroup/ESMValTool/issues/new/choose)
 >     -   If `Solving environment` takes more than 10 minutes, you may need to update
->         conda: `conda update -n base conda`
->     -   You can help conda solve the environment by specifying
+>         mamba: `mamba update -n base mamba`
+>     -   You can help mamba solve the environment by specifying
 >         the python version:
 >         ```bash
->         conda create -n esmvaltool -c conda-forge -c esmvalgroup esmvaltool-python python=3.8
+>         mamba create -n esmvaltool esmvaltool python=3.8
```
Comment (rswamina) [inline _episodes/02-installation.md:229]: Should this be 3.9?

File: _episodes/02-installation.md [hunk 3]
```diff
@@ -221,19 +222,16 @@ installed in it.
 >         the moment, please try again later or [raise an
```
Comment (rswamina) [inline _episodes/02-installation.md:222]: Does it take long now? It was quick for me and I am not aware of any issues. Do you know of any causing delays at this time?

**Review by rswamina — CHANGES_REQUESTED** [review]
I think we can change the voice on the text in the last call out box as follows:
``In my case when I run`` -> ``When you run``

``I get that my installed ESMValTool version is`` -> ``The version of ESMValTool installed should be displayed on the screen as : ``

and we can change versions displayed to 2.4.0 instead of 2.0.0

Let me know what you think.

File: _episodes/02-installation.md (hunk 1 above)
Comment (LisaBock) [reply _episodes/02-installation.md:95]: Thanks. I added that.

File: _episodes/02-installation.md (hunk 2 above)
Comment (LisaBock) [reply _episodes/02-installation.md:229]: Thanks! I changed that.

File: _episodes/02-installation.md (hunk 3 above)
Comment (LisaBock) [reply _episodes/02-installation.md:222]: That was the old list. I deleted it and add a link to the documentation.

**Comment by LisaBock** [comment]
> I think we can change the voice on the text in the last call out box as follows: `In my case when I run` -> `When you run`
> 
> `I get that my installed ESMValTool version is` -> `The version of ESMValTool installed should be displayed on the screen as : `
> 
> and we can change versions displayed to 2.4.0 instead of 2.0.0
> 
> Let me know what you think.

Thanks @rswamina for reviewing!

I made all changes you requested. Let me know if something else needs modifications.

**Review by rswamina — APPROVED** [review]

**Pull request merged**